docs/SBO_User_Guide.md

```diff
@@ -8,31 +8,22 @@
 
 - [Backing Service not providing binding metadata](#Backing-service-not-providing-binding-metadata)
   * [Annotate Service Resources](##Annotate-service-resources)
-  * [Detect Binding Resources](#Detect-binding-resources)
-  * [Compose custom binding variables](#Compose-custom-binding-variables)
+  * [Detect Binding Resources](##Detect-binding-resources)
+  * [Compose custom binding variables](##Compose-custom-binding-variables)
```

Comment:
There is a typo here = `##` it seems it should be just single `#` so that linking can work.

Reply:
Removed SBO from user guide filename.